components/input/input.css

```diff
@@ -0,0 +1,58 @@
+/* Primary Input */
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+}
+
+.input-group {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Suggestion: You can group both classes as both have same properties.

Reply:
Oh yes yes, I'll do that. Thanks!